modules/standard/UtilReplicatedVar.chpl

```diff
@@ -17,27 +17,31 @@
  * limitations under the License.
  */
 
-/*****************************************************************************
-*** SUPPORT FOR USER-LEVEL REPLICATED VARIABLES ***
+/*
+Support for user-level replicated variables.
 
 A "replicated" variable is a variable for which there is a copy on each locale.
 Referencing a replicated variable (in a stylized way, see below)
 accesses its copy on the current locale.
 
 Features:
+
 * The variable's copies are not synchronized automatically among the locales.
 
 Limitations:
+
 * It is "user-level", i.e. the user is required to handle the variable
   in specific ways to achieve the desired result.
+
 * Tree-shape communication (like for reductions) is not provided.
+
 * Using a replicated variable of an array type is not straightforward.
   Workaround: declare that array itself as replicated, then access it normally,
-  e.g.:
-   var replArray: [YOUR DOMAIN dmapped ReplicatedDist()] YOUR ELEMENT TYPE;
+  e.g.::
+
+   var replArray: [MyDomain dmapped ReplicatedDist()] real;
```

Comment:
If you make this a code block (`.. code::` or `.. code-block:: chapel` on a separate line instead of `e.g.::`), you can get syntax highlighting.  See Michael's #1663, for instance.